lib/wpc_proto/config.c

```diff
@@ -0,0 +1,682 @@
+/* Copyright 2019 Wirepas Ltd licensed under Apache License, Version 2.0
```

Comment:
Please update date!

Reply:
Or use the same Copyright as other file from this repo

Reply:
done